app/Models/Mship/Feedback/Answer.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Models\Mship\Feedback;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Answer extends Model
+{
+    protected $table        = 'mship_feedback_answers';
+    protected $dates        = [
+        'created_at',
+        'updated_at',
+    ];
+    protected $fillable     = [
+        'feedback_id',
+        'question_id',
+        'response',
+    ];
+
+    public $timestamps = false;
+
+    public function scopeNotPermanent($query)
+    {
+        return $query->whereHas('question', function ($q) {
+            $q->where('permanent', false);
+        });
+    }
+
+    public function feedback()
+    {
+        return $this->belongsTo('App\Models\Mship\Feedback\Feedback');
```

Comment:
Any class resolutions should be resolved using the 'class' keyword (e.g. \App\Models\Mship\Feedback\Feedback::class) where possible